### DISCUSSION TITLE
Please confirm the URL below is the address of your Homebridge Config UI X instance

### DISCUSSION THREAD

**Discussion by joeferrante** [d0n4v4nb3ck3r/homebridge-honeywell-home#588] [Q&A]
the "Link account" button is not functional when running hoobs behind NPM. when accessing via IP address get a prompt "Please confirm the URL below is the address of your Homebridge Config UI X instance" but the confirm button is not functional.

**Comment by donavanbecker** [comment]
I don't develop this plugin for HOOBS 

I would recommend[ moving to homebridge](https://github.com/homebridge/homebridge-raspbian-image/wiki/How-To-Migrate-From-HOOBS-To-Homebridge).
